Add instahttprouter instrumentation example (#251)
instana/go-sensor: example/README.md

* [gRPC](./grpc-client-server) - an example of usage of Instana tracer in an app instrumented with gRPC

> * [Gin](./gin) - an example of usage of Instana tracer to instrument a Gin application

Reviewer: ```suggestion
* [Gin](./gin) - an example of usage of Instana tracer instrumenting a [`Gin`](github.com/gin-gonic/gin) application
```

Make it the same as in another readme file.

> * [github.com/gorilla/mux](./gorillamux) - an example of usage of Instana tracer to instrument a [`github.com/gorilla/mux`](https://github.com/gorilla/mux) router

Reviewer: ```suggestion
* [Gorilla mux](./gorillamux) - an example of usage of Instana tracer instrumenting the [`github.com/gorilla/mux`](https://github.com/gorilla/mux) router
```
Make it the same as in another readme file.
